src/backend/joanie/payment/backends/base.py

```diff
@@ -27,6 +28,11 @@ class BasePaymentBackend:
 
     name = "base"
 
+    @classproperty
```

Comment:
```suggestion
    @property
```
As you are not using the class itself, you can use the `property` decorator from python I think